app/scripts/controllers/network/network-controller.js

```diff
@@ -246,23 +257,26 @@ export default class NetworkController extends EventEmitter {
     return rpcUrl;
   }
 
-  setRpcTarget(rpcUrl, chainId, ticker = 'ETH', nickname = '', rpcPrefs) {
-    assert.ok(
-      isPrefixedFormattedHexString(chainId),
-      `Invalid chain ID "${chainId}": invalid hex string.`,
-    );
-    assert.ok(
-      isSafeChainId(parseInt(chainId, 16)),
-      `Invalid chain ID "${chainId}": numerical value greater than max safe value.`,
-    );
+  /**
+   * A method for setting a network provider by UUID.
```

Comment:
There are several references to this concept across this file and other files as well but as this is the first I'll call it out here —

I don't think we need to literally call this value "UUID". That ties us to the UUID implementation; if we ever want to change that in the future then we'd have to rename this variable. What if we call it "ID" instead? So:

* "UUID" becomes "ID"
* "network configuration UUID" becomes "network configuration ID"
* `uuid` becomes `id` (except when the context is the provider; then it's `networkConfigurationId` — I'll call this out)
* `configUUID` becomes `networkConfigurationId` (let's try to spell it out as "config" is somewhat of an overloaded term in a controller)

Reply:
Yes this makes sense to me. Keeps it cleaner and more agnostic.